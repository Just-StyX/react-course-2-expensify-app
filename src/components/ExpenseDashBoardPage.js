import React from "react";
import ExpenseList from "./ExpenseList";
import ExpenseListFilters from "./ExpenseListFilters";

const ExpenseDashBoardPage = () => {
  return (
    <div>
      <ExpenseListFilters />
      <ExpenseList />
      <p>This is from my dashboard component</p>
    </div>
  );
};

export default ExpenseDashBoardPage;
