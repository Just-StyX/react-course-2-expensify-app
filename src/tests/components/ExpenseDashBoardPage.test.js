import React from 'react'
import { shallow } from 'enzyme'
import toJSON from 'enzyme-to-json'
import ExpenseDashBoardPage from '../../components/ExpenseDashBoardPage'

test('should render Expense dashboard', () => {
    const wrapper = shallow(<ExpenseDashBoardPage />);
    expect(toJSON(wrapper)).toMatchSnapshot()
})