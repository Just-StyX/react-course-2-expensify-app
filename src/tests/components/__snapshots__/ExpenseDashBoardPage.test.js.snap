// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Expense dashboard 1`] = `
<div>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
  <p>
    This is from my dashboard component
  </p>
</div>
`;
